tests/build_local_test_elfs.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 
 # FILL THESE WITH YOUR OWN SDKs PATHS and APP-ETHEREUM's ROOT
-NANOS_SDK=$NANOS_SDK
```

Comment:
[This script was re-writtent in boilerplate](https://github.com/LedgerHQ/app-plugin-boilerplate/blob/master/tests/build_local_test_elfs.sh) with function and a `set -e` (for early exit on non-zero command results). Maybe it could be useful here?